### DISCUSSION TITLE
"Search panel" is not displayed properly

### DISCUSSION THREAD

**Discussion by AyakaAgo** [community/community#59410] [Accessibility]
### Select Topic Area

Bug

### Body

web page is displayed abnormally after opening the search panel.

![Screenshot_2023-06-27-17-19-49-031_mark via](https://github.com/community/community/assets/20831549/8ad1fe6a-3998-4bac-ad23-fef2962066b7)
![Screenshot_2023-06-27-17-21-46-556_com chrome beta](https://github.com/community/community/assets/20831549/564172ee-f217-442b-a096-65253d27201d)

Chrome version: Chrome Beta 115.0.5790.32
WebView provider: com.google.android.webview 115.0.5790.21(579002142)

**Comment by Belxz777** [comment]
install  github app , or just use other browser
